src/App.js

```diff
@@ -11,6 +9,13 @@ import {
 import { db } from './lib/firebase';
 import { SnackbarProvider } from 'notistack';
 
+import {
+  GlobalStyles,
+  primary,
+  accent,
+  neutral,
+  Layout,
+} from './components/index';
```

Comment:
You don't need to write `/index` at the end of an import, Node will automatically check for a `index` file.

```suggestion
} from './components';
```